plugins/email_service/config/routes.rb

```diff
@@ -1,3 +1,44 @@
 EmailService::Engine.routes.draw do
-  get '/' => 'application#index', as: :index
+
+  get '/' => 'web#index', as: :index
+
+  get 'emails/stats' => 'emails#stats'
+  get '/emails/info' => 'emails#info'
```

Comment:
I didn't found any policy for this route.

Reply:
added policies.